src/utils/v4_position_planner.rs

```diff
@@ -0,0 +1,131 @@
+use super::{
+    to_address,
+    v4_planner::{Actions, V4Planner},
+};
+use crate::{
+    abi::{
+        BurnPositionParams, DecreaseLiquidityParams, IncreaseLiquidityParams, MintPositionParams,
+        SettlePairParams, SweepParams, TakePairParams,
+    },
+    entities::Pool,
+};
+use alloy_primitives::{Address, Bytes, U256};
+use derive_more::derive::{Deref, DerefMut};
```

Comment:
```suggestion
use derive_more::{Deref, DerefMut};
```
Add `derive_more` in Cargo.toml.